frontend/src/lib/Apis.test.ts

```diff
@@ -182,4 +182,16 @@ describe('Apis', () => {
       },
     );
   });
+
+  it('checks if Tensorboard pod is ready', async () => {
+    const spy = jest.fn(() =>
```

Comment:
nit: you can enhance the `fetchSpy` helper with your needed fields.
but I think you should actually check for `response.ok` instead of `response.status == 200`.

Reply:
done